lxc/image.go

```diff
@@ -425,7 +425,7 @@ func (c *imageCmd) run(config *lxd.Config, args []string) error {
 		} else {
 			progress := ProgressRenderer{Format: i18n.G("Transferring image: %s")}
 			handler := func(percent int64, speed int64) {
-				progress.Update(fmt.Sprintf("%d%% (%s/s)", percent, shared.GetByteSizeString(speed)))
+				progress.Update(fmt.Sprintf("%d%% (%s/s)", percent, shared.GetByteSizeString(speed, -1)))
```

Comment:
I agree with @tych0 here. It's an internal function so might as well make things explicit and just pass the precision everywhere.